packaging/bundle-ebpf.sh

```diff
@@ -6,12 +6,14 @@ PLUGINDIR="${2}"
 EBPF_VERSION="$(cat "${SRCDIR}/packaging/ebpf.version")"
 EBPF_TARBALL="netdata-kernel-collector-glibc-${EBPF_VERSION}.tar.xz"
 
-mkdir -p "${SRCDIR}/tmp/ebpf"
-curl -sSL --connect-timeout 10 --retry 3 "https://github.com/netdata/kernel-collector/releases/download/${EBPF_VERSION}/${EBPF_TARBALL}" > "${EBPF_TARBALL}" || exit 1
-grep "${EBPF_TARBALL}" "${SRCDIR}/packaging/ebpf.checksums" | sha256sum -c - || exit 1
-tar -xaf "${EBPF_TARBALL}" -C "${SRCDIR}/tmp/ebpf" || exit 1
-if [ ! -d "${PLUGINDIR}/ebpf.d" ];then
-    mkdir "${PLUGINDIR}/ebpf.d"
+if [ -x "${PLUGINDIR}/ebpf.plugin" ] ; then
```

Comment:
Should we print an info message if this condition is not true?

Reply:
I think we’re fine for now without it.